tests/e2e/pact/services/get-access-token.ts

```diff
@@ -1,7 +1,7 @@
 import {createJWT} from "./jwt"
 import axios from "axios"
 import {URLSearchParams} from "url"
-import * as fs from "fs"
+import fs from "fs"
 import path from "path"
 
 /* TODO - get from ADO / env variables */
```

Comment:
Do we want to pickup this todo as part of this pr, or next?

Reply:
ideally neither lol its work that won't really help us with epsat so would rather not spend time on it. this is just a script to print a token to console, only works for internal-dev atm